server/start.go

```diff
@@ -244,18 +298,16 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 	err = svr.Start()
 	if err != nil {
 		fmt.Println(err.Error())
-		os.Exit(1)
+		return err
 	}
 
 	defer func() {
-		if err = svr.Stop(); err != nil {
-			fmt.Println(err.Error())
-			os.Exit(1)
-		}
+		_ = svr.Stop()
```

Comment:
To handle this similarly to the case with the in-process server, I changed these to even stop the app if there is an error here